### ISSUE TITLE
Showing this error how to solve it ?

### ISSUE THREAD

**Issue by gauravkumarjha442** [rishz/URL-Shortener#202]
npm ERR! code ELIFECYCLE npm ERR! errno 3221226505 npm ERR! urlshortener@1.0.0 start: `node server.js` npm ERR! Exit status 3221226505 npm ERR! npm ERR! Failed at the urlshortener@1.0.0 start script. npm ERR! This is probably not a problem with npm. There is likely additional logging output above. npm ERR! A complete log of this run can be found in: npm ERR! C:\Users\GAURAV KUMAR JHA\AppData\Roaming\npm-cache\_logs\2021-01-29T19_47_03_491Z-debug.log

**Comment by imtiyaj786** [comment]
You have to  Install Node.js and NPM from Browser. by using this command nmp is install   npm install -g npm

**Comment by gauravkumarjha442** [comment]
> You have to Install Node.js and NPM from Browser. by using this command nmp is install npm install -g npm

thanks , Now not showing any error . But when run this using  "npm start" cmd , after some time it auto terminated,what to do for that?

**Comment by gauravkumarjha442** [comment]
E:\MyProjects\Git_Github\URL-Shortener>npm start

> c@1.0.0 start
> node server.js

Listening on port 4100

E:\MyProjects\Git_Github\URL-Shortener>

**Comment by imtiyaj786** [comment]
Yes when you run this command its work from spacific time. So when its terminated you have to do same "npm start"

**Comment by gauravkumarjha442** [comment]
ok thanks,
